10_Getting_Started/20_Build/10_Linux.md

```diff
@@ -99,6 +99,18 @@ SOFA requires some libraries:
     Make sure to enable **Charts** and **WebEngine** components.  
     ![](https://www.sofa-framework.org/wp-content/uploads/2020/04/install_qt_linux.png)
 
+    Install the Qt Wayland Plugin
+
+    ```bash
+    sudo apt install qtwayland5
```

Comment:
well  because it happens that I am using qt 5.15, so i think that is  why i encountered the problem so i don't think its necessary for Qt6 .
So should i remove it and only keep the command to     Set the Environment Variable